### ISSUE TITLE
Syncout TypeError on mobile messages with newlines

### ISSUE THREAD

**Issue by BIGfin86** [hangoutsbot/hangoutsbot#473]
Receive this error when sending messages from android app that contains newline/line break.  These messages error out and are not relayed across the sync.  Able to reproduce on a few bots.

Example message that wont sync being sent from mobile app:
>This
>message will not
>sync

Hangoutbot version *2.7.4*
Hangouts Android Version *6.1.109448852*

```
ERROR handlers: message:plugins.syncrooms._handle_incoming_message : coroutine
Traceback (most recent call last):
  File "/home/bot/hangoutsbot/hangupsbot/handlers.py", line 256, in run_pluggable_omnibus
    yield from function(*_passed)
  File "/home/bot/hangoutsbot/hangupsbot/plugins/syncrooms.py", line 153, in _handle_incoming_message
    source_message += segment.text
TypeError: Can't convert 'NoneType' object to str implicitly
```

**Comment by gurumadmat** [comment]
Fixed in eedb12df3cbe0b4f0f17987c750bca9217dae6ab.

**Comment by nylonee** [comment]
See #723 and 26087aab23287642242f9ae249b5ec7b9202995a for a better fix
